### DISCUSSION TITLE
Release notes for January 9

### DISCUSSION THREAD

**Discussion by mcoughlin** [skyportal/skyportal#3828] [Announcements]
First release of 2023 with many contributions from ESLIV students!

## Front end
📑 Download observation tables with RA and Dec [PR #3805] (thanks @mcoughlin)
📊 Modify photometry form to enable single point upload [PR #3805 and PR #3808] (thanks @Theodlz / @mcoughlin )
💹 Fix instrument table to use MUI for searchability [PR #3815] (thanks @lydie10 and @Carla802)
👍 Only display classification votes if requested [PR #3823] (thanks @mcoughlin)
🌞 Keep history of GCN Summaries  [PR #3823] (thanks @jolyanem and @Nayru12)

## Back end
☎️ Notify on-shift users of follow-up requests [PR #3807] (thanks @mcoughlin)
📞 Enable notifications from WhatsApp [PR #3814] (thanks @aolrc-vinci and @ManonFlt)

## Bug fixes
:lady_beetle: Clear source table filter form when parameters are inconsistent [PR #3810] (thanks @KodeurFou and @Adiman007)

**Comment by Theodlz** [comment]
Congrats to all the new students 👏
